File: PooCC01CNetz/Semana03/Python/Ejercicios2/Vector3D.py
import math

class Vector3D:
    def __init__(self, refX = 0.0, refY = 0.0, refZ = 0.0):
        self.refX = refX
        self.refY = refY
        self.refZ = refZ

    def setRefX(self, refX):
        self.refX = refX

    def getRefX(self):
        return self.refX

    def setRefY(self, refY):
        self.refY = refY

    def getRefY(self):
        return self.refY

    def setRefZ(self, refZ):
        self.refZ = refZ

    def getRefZ(self):
        return self.refZ

    def MagnitudVector(self):
        mag = math.sqrt(self.refX * self.refX + self.refY * self.refY + self.refZ * self.refZ)
        return mag

File: PooCC01CNetz/Semana03/Java/Ejercicios1/Vector3D.java
public class Vector3D {
    private double refX;
    private double refY;
    private double refZ;

    public Vector3D(double refX, double refY, double refZ) {
        this.refX = refX;
        this.refY = refY;
        this.refZ = refZ;
    }

    public Vector3D() {}

    public void setRefX(double refX) {
        this.refX = refX;
    }

    public double getRefX() {
        return refX;
    }

    public void setRefY(double refY) {
        this.refY = refY;
    }

    public double getRefY() {
        return refY;
    }

    public void setRefZ(double refZ) {
        this.refZ = refZ;
    }

    public double getRefZ() {
        return refZ;
    }

    public double MagnitudVector(){
        double mag = Math.sqrt(refX * refX + refY * refY + refZ * refZ);
        return mag;
    }
}